Kalela/Level-Up
# #155965773 Github users adapter

# What does this PR do?
Creates an adapter ``` GithubUsersAdapter.java ``` that leverages the retrofit instance to render Java developers in Nairobi using the github api.
### Screenshots
![ezgif com-optimize](https://user-images.githubusercontent.com/33759757/42450463-ebe2e2f4-838c-11e8-8beb-cdf36636c916.gif)


### How should this be manually tested?
- Clone the repo.
- Open the project in android studio.
- Run the project using an emulator or adb.
- Try searching for developers shown on Github to confirm they are from github api.



### What are the relevant pivotal tracker stories?
#### 155965773
Create the GithubUsersAdapter

#### 155965831
Display List of Github Users
